files/sanity_check.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
This file doesn't have tests currently? I know we are not testing the ansible scripts themselves, but it might be worth testing this one